src/test/ui-fulldeps/lint-plugin-forbid-attrs.stderr

```diff
@@ -21,4 +21,4 @@ error[E0453]: allow(test_lint) overruled by outer forbid(test_lint)
 
 error: aborting due to 2 previous errors
 
-If you want more information on this error, try using "rustc --explain E0453"
+For more information about this error, try "rustc --explain E0453"
```

Comment:
`"rustc --explain E0453"` -> `` `rustc --explain E0453` ``

Reply:
Since we start with a capital letter (seems appropriate here) then we should probably end with a dot.